tests/installation/bootloader_uefi.pm

```diff
@@ -33,6 +34,13 @@ sub run() {
     if (get_var("QEMUVGA") && get_var("QEMUVGA") ne "cirrus") {
         sleep 5;
     }
+    if (is_jeos) {
+        # tell grub to use the correct gfx mode (bnc#963952)
+        send_key 'c';
+        type_string "gfxmode=1024x768; terminal_output console; terminal_output gfxterm\n";
+        sleep 2;
```

Comment:
would you mind using "wait_still_screen" instead? Also the "type_string" should maybe be slowed down. Maybe merge https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/1076 first? I don't want to merge my own PR without approval